applications/client/src/main/java/com/paritytrading/philadelphia/client/command/WaitCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.paritytrading.philadelphia.client.command;
+
+import com.paritytrading.philadelphia.client.TerminalClient;
+import com.paritytrading.philadelphia.client.message.Message;
+import com.paritytrading.philadelphia.client.message.Messages;
+import org.eclipse.collections.api.list.ImmutableList;
+
+import java.util.Scanner;
```

Comment:
In Philadelphia, the imports are usually grouped together in alphabetical order. 🤓